Model/src/main/java/org/gusdb/wdk/model/user/dataset/event/UserDatasetEventCleanup.java

```diff
@@ -0,0 +1,81 @@
+package org.gusdb.wdk.model.user.dataset.event;
+
+import javax.sql.DataSource;
+
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+import org.gusdb.wdk.model.WdkModelException;
+import org.gusdb.wdk.model.user.dataset.UnsupportedTypeHandler;
+import org.gusdb.wdk.model.user.dataset.event.model.UserDatasetUninstallEvent;
+
+public class UserDatasetEventCleanup extends UserDatasetEventProcessor
```

Comment:
seems to me this class should have a comment describing the overall strategy of this class

Reply:
Added doc block.